django-stubs/core/files/uploadedfile.pyi

```diff
@@ -44,7 +44,9 @@ class InMemoryUploadedFile(UploadedFile):
     ) -> None: ...
     def open(self, mode: str | None = ...) -> Self: ...  # type: ignore[override]
 
+_FileDict = TypedDict("_FileDict", {"filename": str, "content": bytes | None, "content-type": NotRequired[str])
```

Comment:
Please, use class syntax.

Reply:
inline syntax used because of the `content-type` key.